src/OpenSpaceToolkit/Physics/Environment/Ephemerides/SPICE/Engine.cpp

```diff
@@ -13,7 +13,7 @@
 #include <OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/Dynamic.hpp>
 
 // clang-format off
-// Include Manager.hpp before Engine.hpp to avoid name conflict for "ostk::core::types::Index" 
+// Include Manager.hpp before Engine.hpp to avoid name conflict for "ostk::core::types::Index"
 #include <OpenSpaceToolkit/Physics/Environment/Ephemerides/SPICE/Manager.hpp>
 #include <OpenSpaceToolkit/Physics/Environment/Ephemerides/SPICE/Engine.hpp>
```

Comment:
I believe instead of this, we can merely use an alias in Engine. IMO that's a better solution as there is an actual conflict here so it's safer to use an alias like
```
using IndexType = ostk::core::types::Index
```